src/ucp/wireup/address.c

```diff
@@ -396,12 +396,12 @@ ucp_address_unpack_iface_attr(ucp_worker_t *worker,
         unified               = ptr;
         rsc_idx               = unified->rsc_index & UCP_ADDRESS_FLAG_LEN_MASK;
         iface_attr->lat_ovh   = fabs(unified->lat_ovh);
-        wiface                = ucp_worker_iface(worker, rsc_idx);
-        ucs_assertv_always(rsc_idx < worker->context->num_tls,
-                           "rsc_idx=%u num_tls=%u",
-                           rsc_idx , worker->context->num_tls);
+        ucs_assertv_always(UCS_BIT(rsc_idx) & worker->context->tl_bitmap,
+                           "rsc_idx=%u tl_bitmap=0x%"PRIx64,
+                           rsc_idx , worker->context->tl_bitmap);
```

Comment:
extra whitespace after `rsc_idx`

Reply:
also, can you pls do (for consistency)
`UCS_BIT(rsc_idx) & worker->context->tl_bitmap` -> `worker->context->tl_bitmap & UCS_BIT(rsc_idx)`
we do it in this way in other places